ddtrace/bootstrap/sitecustomize.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import print_function
+
+"""
+Bootstrapping code that is run when using the `ddtrace-run` Python entrypoint
+Add all monkey-patching that needs to run by default here
+"""
+
+try:
+    from ddtrace import patch_all; patch_all(django=True, flask=True, pylons=True) # noqa
```

Comment:
`django, pylons, flask` are not patched by default because we want `patch_all()` to continue to play nice with the existing way of instrumenting web frameworks